kodilla-rps/src/main/java/com/kodilla/rps/round/RoundChecker.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.rps.round;
+
+import com.kodilla.rps.model.Shape;
+
+public class RoundChecker {
+
+    private Shape compare(Shape userShape, Shape computerShape) {
```

Comment:
Metoda nie może zwracać Shape. Moim zdaniem trzeba wprowadzić enum `RoundResult` i to tutaj zwrócić.